mqtt/api/transport.go

```
@@ -0,0 +1,82 @@
+		}
+		res, err := clients.Authenticate(r.Context(), &grpcClientsV1.AuthnReq{ClientSecret: password})
+		if err != nil {
+			http.Error(w, errors.Wrap(svcerr.ErrAuthentication, err).Error(), http.StatusBadRequest)
+			return
+		}
+		if !res.GetAuthenticated() {
+			http.Error(w, svcerr.ErrAuthentication.Error(), http.StatusBadRequest)
+			return
+		}
+
+		w.Write([]byte("allow"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)